elements/lisk-codec/src/boolean.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+/* eslint-disable no-bitwise */
+/* eslint-disable no-param-reassign */
+
+
+interface SchemaProperty {
+	readonly dataType: string;
```

Comment:
boolean?

Reply:
`dataType` itself is `string` and it contains `"boolean"`